app/controllers/pieces_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PiecesController < ApplicationController
+	def update
+    @piece = Piece.find(params[:id])
+    @game = @piece.game
+    @piece.move_to!(piece_params)
```

Comment:
Oh, wait, ignore that - this is an AJAX thing.  You probably want to respond with a success / fail message like so:

```ruby
# either
render json: 'success'

# or
render json: 'failure'
```

That way the JS on the frontend can let the user know if it worked.